packages/common/src/hardhat/tasks/compile.js

```diff
@@ -33,16 +35,11 @@ internalTask(TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS, async (_, { config, network
       return false;
     });
   }
-  // Temp work around to exclude some directories from truffle while compiling
-  // with hardhat.
-  const ovmPath = path.resolve(__dirname, "../../../../core/contracts-ovm");
-
-  return [
-    ...filePaths,
-    `${ovmPath}/insured-bridge/implementation/L1_BridgePool.sol`,
-    `${ovmPath}/insured-bridge/implementation/L1_BridgeRouter.sol`,
-    `${ovmPath}/insured-bridge/implementation/L2_BridgeDepositBox.sol`,
-  ];
-
-  // return filePaths;
+  // Temp work around to exclude some directories from truffle while compiling with hardhat. Pulls in all contracts in
```

Comment:
This will have a merge conflict with #3256 because I put this entire section within an `else {}` statement. See [this](https://github.com/UMAprotocol/protocol/pull/3256#discussion_r678291803) comment.

Reply:
ye. it's ok. merge your code in first and I'll rebase and resolve the conflict tomorrow. Your PR is through CI and mine is still blocked.